### ISSUE TITLE
Cannot run vscode or run tests after building from source code

### ISSUE THREAD

**Issue by jarl-dk** [microsoft/vscode#116922]
- VS Code Version: 1.52.1 (git commit ea3859d4ba2f3e577a159bc91e3074c5d85c0523)
- OS Version: Ubuntu 20.04.2

Steps to Reproduce:

1. clone repository and checkout tag `1.52.1`
2. Build (following https://github.com/microsoft/vscode/wiki/How-to-Contribute#build-and-run), run
```
yarn
```
3. Run the unit tests (following https://github.com/microsoft/vscode/blob/main/test/unit/README.md):
```
./scripts/test.sh
```
4. Run vscode
```
./scripts/code.sh --disable-extensions
```

Does this issue occur when all extensions are disabled?: Yes/No

The problem occurs both at step 3 and 4:

At step 3 (unit tests) the result is:
```
$ scripts/test.sh 
yarn run v1.22.4
$ node build/lib/electron
Done in 0.78s.
[24580:0218/065743.167273:INFO:CONSOLE(221)] "{"code":"MODULE_NOT_FOUND","requireStack":["/home/jarl/projects/vscode/test/unit/electron/renderer.js","/home/jarl/projects/vscode/test/unit/electron/renderer.html"],"phase":"loading","moduleId":"vscode-minimist","neededBy":["vs/platform/environment/node/argv"]}", source: /home/jarl/projects/vscode/test/unit/electron/renderer.js (221)
{"code":"MODULE_NOT_FOUND","requireStack":["/home/jarl/projects/vscode/test/unit/electron/renderer.js","/home/jarl/projects/vscode/test/unit/electron/renderer.html"],"phase":"loading","moduleId":"vscode-minimist","neededBy":["vs/platform/environment/node/argv"]}
```
At step 4 (running vscode) the result is:
```
$ scripts/code.sh --disable-extensions
yarn run v1.22.4
$ node build/lib/electron
Done in 0.79s.
[06:58:27] Syncronizing built-in extensions...
[06:58:27] You can manage built-in extensions with the --builtin flag
[06:58:27] [marketplace] ms-vscode.node-debug@1.44.15 ✔︎
[06:58:27] [marketplace] ms-vscode.node-debug2@1.42.5 ✔︎
[06:58:27] [marketplace] ms-vscode.references-view@0.0.74 ✔︎
[06:58:27] [marketplace] ms-vscode.js-debug-companion@1.0.8 ✔︎
[06:58:27] [marketplace] ms-vscode.js-debug@1.52.2 ✔︎
[06:58:27] [marketplace] ms-vscode.vscode-js-profile-table@0.0.11 ✔︎
[06:58:27] [marketplace] ms-vscode.github-browser@0.0.14 ✔︎
App threw an error during load
Error: Cannot find module 'vscode-minimist'
Require stack:
- /home/jarl/projects/vscode/out/main.js
- /home/jarl/projects/vscode/.build/electron/resources/default_app.asar/main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:797:17)
    at Function.o._resolveFilename (electron/js2c/browser_init.js:281:681)
    at Module._load (internal/modules/cjs/loader.js:690:27)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at parseCLIArgs (/home/jarl/projects/vscode/out/main.js:194:19)
    at Object.<anonymous> (/home/jarl/projects/vscode/out/main.js:29:14)
    at Module._compile (internal/modules/cjs/loader.js:967:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1004:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at loadApplicationPackage (/home/jarl/projects/vscode/.build/electron/resources/default_app.asar/main.js:109:16)
    at Object.<anonymous> (/home/jarl/projects/vscode/.build/electron/resources/default_app.asar/main.js:155:9)
    at Module._compile (internal/modules/cjs/loader.js:967:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1004:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Object.<anonymous> (electron/js2c/browser_init.js:205:3373)
    at Object../lib/browser/init.ts (electron/js2c/browser_init.js:205:3577)
    at __webpack_require__ (electron/js2c/browser_init.js:1:128)
    at electron/js2c/browser_init.js:1:1200
    at electron/js2c/browser_init.js:1:1267
    at NativeModule.compile (internal/bootstrap/loaders.js:287:5)
    at NativeModule.compileForPublicLoader (internal/bootstrap/loaders.js:222:8)
    at loadNativeModule (internal/modules/cjs/helpers.js:23:9)
    at Module._load (internal/modules/cjs/loader.js:698:15)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1056:10)
    at internal/main/run_main_module.js:17:11
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'vscode-minimist'
Require stack:
- /home/jarl/projects/vscode/out/main.js
- /home/jarl/projects/vscode/.build/electron/resources/default_app.asar/main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:797:17)
    at Function.o._resolveFilename (electron/js2c/browser_init.js:281:681)
    at Module._load (internal/modules/cjs/loader.js:690:27)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at parseCLIArgs (/home/jarl/projects/vscode/out/main.js:194:19)
    at Object.<anonymous> (/home/jarl/projects/vscode/out/main.js:29:14)
    at Module._compile (internal/modules/cjs/loader.js:967:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1004:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at loadApplicationPackage (/home/jarl/projects/vscode/.build/electron/resources/default_app.asar/main.js:109:16)
    at Object.<anonymous> (/home/jarl/projects/vscode/.build/electron/resources/default_app.asar/main.js:155:9)
    at Module._compile (internal/modules/cjs/loader.js:967:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1004:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Object.<anonymous> (electron/js2c/browser_init.js:205:3373)
    at Object../lib/browser/init.ts (electron/js2c/browser_init.js:205:3577)
    at __webpack_require__ (electron/js2c/browser_init.js:1:128)
    at electron/js2c/browser_init.js:1:1200
    at electron/js2c/browser_init.js:1:1267
    at NativeModule.compile (internal/bootstrap/loaders.js:287:5)
    at NativeModule.compileForPublicLoader (internal/bootstrap/loaders.js:222:8)
    at loadNativeModule (internal/modules/cjs/helpers.js:23:9)
    at Module._load (internal/modules/cjs/loader.js:698:15)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1056:10)
    at internal/main/run_main_module.js:17:11
^C
```

**Comment by jarl-dk** [comment]
The problem also exists on latest main.

**Comment by joaomoreno** [comment]
Can you clean and `yarn` again, it just seems you do not have all dependencies installed.

```
git clean -xfd .
yarn
```

**Comment by jarl-dk** [comment]
I realise now that by following [the link"How to build and run from source" from the README.md](https://github.com/microsoft/vscode/blame/main/README.md#L33) directing me to https://github.com/microsoft/vscode/wiki/How-to-Contribute#build-and-run I ended up on an anchor on the middle of a page after describing how to set up the development environment. I completely missed the prerequisites by following that link. I have now installed the necessary libraries, clean up with
```
git clean -xfd
yarn cache clean
```
and then ran
```
yarn
```
Then I can run `./scripts/code.sh` without problems.
I suggest you change the [the link"How to build and run from source" from the README.md](https://github.com/microsoft/vscode/blame/main/README.md#L33) to go to the top of the page.

Thanks for help
